src/components/List/index.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useState } from 'react';
+import classNames from 'classnames';
+import isEqual from 'lodash.isequal';
+
+import styles from '@/components/List/List.css';
+
+import ListItem from '@/components/ListItem';
+
+type ListItemCategory = 'simple' | 'icon' | 'checkbox';
+type Size = 'large' | 'medium';
+
+const largeSize = 'large';
+
+interface Props<T> {
+  getItemIsSelected?: (item?: T) => boolean;
```

Comment:
if you are expecting a boolean then it is not a get, but just a question: `isItemSelected`